cinema/views.py

```diff
@@ -56,3 +102,20 @@
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+    pagination_class = OrderPagination
+
+    def get_queryset(self):
+        return self.queryset.filter(user=self.request.user)
```

Comment:
The `get_queryset` method assumes that the user is authenticated. Ensure that this viewset is protected by authentication and permission classes to prevent unauthorized access.